src/main/groovy/org/labkey/gradle/plugin/FileModule.groovy

```diff
@@ -168,10 +170,30 @@ class FileModule implements Plugin<Project>
                             String newLine = line
                             while (matcher.find())
                             {
-                                newLine = newLine.replace(matcher.group(), (String) project.lkModule.getPropertyValue(matcher.group(1), ""))
+                                if(PropertiesUtils.LABKEY_VERSION_PATTERN.matcher(line))
```

Comment:
I think this doesn't need to be as specialized as this.  If we add in logic to get this property to the ModuleExtension.setModuleProperties method, then the logic here can stay the same.  And, this property value should be available from the project itself (`project.getProperty('labkeyVersion')`), so there shouldn't be a need to parse this file ourselves.

Reply:
Awesome! Didn't know about the labkey versino already existing in the project object. Much simpler code now.